pkgs/development/python-modules/airtouch4pyapi/default.nix

```diff
@@ -16,8 +16,8 @@ buildPythonPackage rec {
     owner = "LonePurpleWolf";
     repo = pname;
     # https://github.com/LonePurpleWolf/airtouch4pyapi/issues/5
-    rev = "34783888846783c058fe79cec16feda45504f701";
-    sha256 = "17c7fm72p085pg9msvsfdggbskvm12a6jlb5bw1cndrqsqcrxywx";
+    rev = "5b5d91fad63495c83422e7a850897946ac95b25d";
+    hash = "sha256-lhKFsXbyEgx8+0h37kunV3IYZwcHGJCyFElzgxH/iZE=";
```

Comment:
Can we apply this as a patch? Making the change this way is quite intransparent.

Reply:
We can, but is that really so much better given that we're already fetching a random commit (which happens to be HEAD right now).

Reply:
This make src the upstream version and the patch indicates something is not alright upstream, which is … a statement.

Reply:
But it *is* upstream, just only on PyPI.

Reply:
Yeah, very weird all things considered.